scripts/installscripts/stretch-install-default.sh

```diff
@@ -410,6 +410,7 @@ git fetch
 
 # Install more required packages
 sudo pip install -r requirements.txt
+sudo pip3 install -r /home/pi/RPi-Jukebox-RFID/components/rfid-reader/rfid-reader-PN532/requirements-pn532.txt
```

Comment:
I would not add the installation to the scripts for stretch.

Stretch is still much Python 2 code and I would expect some problems on stretch. If anyone wants to use that reader on stretch, it's still possible to install it manually

Reply:
> Stretch is still much Python 2 code and I would expect some problems on stretch.

A few lines above (https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/scripts/installscripts/stretch-install-default.sh#L396) Python3 is installed. This seems to be inconsistent with your statement. I also tested it with Python3 on Stretch without any issues.

I think the way how additional components should be installed needs to be improved anyway, but until that is tackled, I'd recommend to make the install scripts as similar as possible and avoid having to document different behaviors.

Reply:
You're right, python3 is used in stretch as well, but there really wasn't much testing lately.

Users are encouraged to use buster for new installations (e.g. mopidy 3 for Spotify edition won't work with stretch anymore). 

So we don't really get any feedback, if new installations on stretch still work.

Nevertheless I don't see much harm, if stretch also installs this reader.